packages/gatsby-theme-api-docs/src/components/resource/method/parameters.js

```diff
@@ -77,15 +85,37 @@ function ParameterRow(props) {
       <td>
         <PropertyName>
           <SpacingsInline scale="xs">
-            <Markdown.InlineCode>{props.parameter.name}</Markdown.InlineCode>
+            {isRegex(props.parameter.name) ? (
+              <Markdown.InlineCode>
+                <span
+                  css={css`
+                    color: ${uiKitDesignSystem.colors.light.textInfo};
+                  `}
+                >
+                  /
+                </span>
+                {getExpressionInsideSlashes(props.parameter.name)[1]}
+                <span
+                  css={css`
+                    color: ${uiKitDesignSystem.colors.light.textInfo};
+                  `}
+                >
+                  /
+                </span>
+              </Markdown.InlineCode>
```

Comment:
Same here, this is 100% duplicate.  Did you consider making it a component that can be used by both?